core/src/main/java/io/grpc/PickFirstLoadBalancerFactory.java

```diff
@@ -42,30 +42,29 @@
 import javax.annotation.concurrent.GuardedBy;
 
 /**
- * A {@link LoadBalancer} that provides simple round-robin and pick-first routing mechanism over the
+ * A {@link LoadBalancer} that provides pick-first routing mechanism over the
  * addresses from the {@link NameResolver}.
  */
-// TODO(zhangkun83): Only pick-first is implemented. We need to implement round-robin.
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1771")
-public final class SimpleLoadBalancerFactory extends LoadBalancer.Factory {
+public final class PickFirstLoadBalancerFactory extends LoadBalancer.Factory {
```

Comment:
I used this name because it's what the c version was called.  I'm happy to name it anything though, or even revert to Simple, since that's a lot like Dummy even.